docker-compose.override.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.7'
+services:
+  database:
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_USER: root
+      MYSQL_PASSWORD: root
+      MYSQL_DATABASE: testdb
+  web:
+    environment:
+      DB_URL: mysql://root:root@database:3306/testdb
+      DB_NAME: testdb
+      DB_SYNCHRONIZE: 'true'
```

Comment:
Have you set `DB_SYNCHRONIZE` to false in production? Since it should be only be `true` when developing :) And btw how this file ended up in this pr :D

Reply:
It is false in production, I've always made the changes through dev environment using the synchronize though! I had a feeling there was a better way for doing this though but since I never was afraid of losing data, I didn't put too much thought to it!

The reason this file is here is because I accidentally started the branch from very old version of master and had to rebase! :D

Reply:
Hmm but if the rebase was succesful then you should be able to remove this file? :D